src/patternfly/components/DualListSelector/dual-list-selector.hbs

```diff
@@ -1,4 +1,4 @@
-<div class="pf-c-dual-list-selector{{#if dual-list-selector--modifier}} {{dual-list-selector--modifier}}{{/if}}"
+<div class="pf-c-dual-list-selector{{#if dual-list-selector--IsTreeview}} pf-m-tree-view{{/if}}{{#if dual-list-selector--modifier}} {{dual-list-selector--modifier}}{{/if}}"
```

Comment:
Is this class necessary, or could we scope the style changes to `__list-item.pf-m-expandable`?

Reply:
removed